core/sdk-core/src/main/java/software/amazon/awssdk/core/interceptor/Context.java

```diff
@@ -105,15 +144,16 @@ private Context() {}
      */
     @ThreadSafe
     @SdkPublicApi
-    public interface ModifyHttpResponse extends AfterTransmission {
+    public interface ModifyHttpResponseContent extends ModifyAsyncHttpResponseContent {
+        Publisher responsePublisher();
     }
```

Comment:
No raw generics, please.
Shouldn't this be in AfterTransmision?
Where is the sync payload?